### DISCUSSION TITLE
Design tokens and Tailwind config

### DISCUSSION THREAD

**Discussion by o-t-w** [tailwindlabs/tailwindcss#9493] [Ideas]
Design tokens have recently been standardized. https://design-tokens.github.io/community-group/format/
I see the use of design tokens becoming more common now that they are standardized.
It would be great it there was a default and easy way to utilize them with Tailwind.

**Comment by morozovamv** [comment]
+1 to the topic, it's hard to understand how to connect tokens to the tailwind config. Colors are possible to implement, but the rest is tricky.
It seems that different fields have different logic of handling values, for example if I put `{ my: { custom: { path: { value: 'something' } } } }` to the `config.theme.extend.colors`, then I can use it like this: `tw"bg-my-custom-path-value"`; but if I do the same with the `padding`, it doesn't work at all.
Can't find any docs that would describe correct way of extending fields.
